spec/controllers/home_controller_spec.rb

```diff
@@ -3,8 +3,8 @@
 RSpec.describe HomeController, type: :controller do
   describe 'GET #index' do
     before(:each) do
-      stub_request(:get, /redmine.renuo.ch/)
-        .to_return(status: 200, body: 'Test', headers: {})
+      http_basic_auth
+      stub_request(:get, /redmine.renuo.ch/).to_return(status: 200, body: 'Test', headers: {})
     end
```

Comment:
probably we have to test what happens if the we are not logged in. 

context 'user is logged in' do
and 
context 'user is not logged in' do